datacite-delete-draft-doi-handler/src/main/java/no/unit/nva/datacite/handlers/resource/DoiEntity.java

```diff
@@ -0,0 +1,5 @@
+package no.unit.nva.datacite.handlers.resource;
+
+import java.net.URI;
+
+public record DoiEntity(String identifier, URI doi, Publisher publisher) {}
```

Comment:
Identifier here is resourceIdentifier, right?

Maybe call it `resourceIdentifier` and add `@JsonProperty("identifier")` annotation

Reply:
There is only one identifier we should be using here and it is a URI…we push different environment's data to test

Reply:
Ideally, yes. But as this is effectively the Resource entity (publication dao), the id is not present here. Identifier is only used for logging to trace reason for drafted doi deletion.